spec/requests/distributions_by_county_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "DistributionsByCounties", type: :request do
   let(:default_params) do
-    {organization_name: @organization.to_param}
+    {organization_name: organization.to_param}
```

Comment:
This is doing more than just adding the `skip_seed`... no?

Reply:
https://github.com/rubyforgood/human-essentials/pull/4273

These changes need to go in for the change to work. This PR is rebased on top of another pr

Reply:
If this is blocked on another PR, can you move it to draft please?